packages/edge-login-ui-rn/src/native/components/abSpecific/OtpHeroComponent.js

```diff
@@ -33,12 +34,13 @@ class OtpHeroComponent extends Component<Props, State> {
       return (
         <View style={style.orOption}>
           <View style={style.orRow}>
-            <Text style={style.instructionsText}>──────── OR ────────</Text>
+            <Text style={style.instructionsText}>
+              ──────── {s.strings.or} ────────
+            </Text>
```

Comment:
This could get interesting in other languages if the word "OR" changes length a lot. Would the dashes make more sense in the string, or should we cross that bridge when we get to it?